<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #5b5b5b;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="change_state">
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    color: white;
	text-align: 10px;
}

QCheckBox::indicator {
    background-color: #333; 
    border: 1px solid #555; 
    width: 15px; 
    height: 15px; 
}

QCheckBox::indicator:checked {
    background-color: #008080;
}</string>
      </property>
      <property name="text">
       <string>Включить/выключить авто-кликер</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #484848;
border: 2px solid #3e3e3e;
border-radius: 5px;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="title">
          <string>Настроить состояние клавиш</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="lbutton_state">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox {
	border: none;
    color: white;
}

QCheckBox::indicator {
    background-color: #333; 
    border: 1px solid #555; 
    width: 15px; 
    height: 15px; 
}

QCheckBox::indicator:checked {
    background-color: #008080;
}</string>
            </property>
            <property name="text">
             <string>Вкл/выкл левую кнопку мыши</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rbutton_state">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox {
	border: none;
    color: white;
}

QCheckBox::indicator {
    background-color: #333; 
    border: 1px solid #555; 
    width: 15px; 
    height: 15px; 
}

QCheckBox::indicator:checked {
    background-color: #008080;
}</string>
            </property>
            <property name="text">
             <string>Вкл/выкл правую кнопку мыши</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>165</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>165</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #484848;
border: 2px solid #3e3e3e;
border-radius: 5px;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="state_own_wait">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body style=&quot;border:none;&quot;&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff; border:none;&quot;&gt;Устанавливает индивидуальную задержку между нажатиями, которую указываете вы сами&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox {
	border: none;
    color: white;
}

QCheckBox::indicator {
    background-color: #333; 
    border: 1px solid #555; 
    width: 15px; 
    height: 15px; 
}

QCheckBox::indicator:checked {
    background-color: #008080;
}</string>
         </property>
         <property name="text">
          <string>Вкл/выкл свою задержку (?)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="title">
          <string>Укажите задержку</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="0">
           <widget class="QScrollBar" name="rbutton_wait_slider">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QScrollBar:horizontal {
    border: 1px solid #999999;
    background: #f0f0f0;
    height: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:horizontal {
    background: #666666;
    border: 1px solid #666666;
    min-width: 20px;
}

QScrollBar::add-line:horizontal {
    border: 1px solid #999999;
    background: #f0f0f0;
    width: 0px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
    border: 1px solid #999999;
    background: #f0f0f0;
    width: 0px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::add-page:horizontal,
QScrollBar::sub-page:horizontal {
    background: #4b4b4b;
}

QScrollBar::left-arrow:horizontal,
QScrollBar::right-arrow:horizontal {
    border: 1px solid #999999;
    width: 3px;
    height: 3px;
    background: #f0f0f0;
}
</string>
            </property>
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string>Задержка ЛКМ</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lbutton_wait_label">
            <property name="text">
             <string>мс</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QScrollBar" name="lbutton_wait_slider">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QScrollBar:horizontal {
    border: 1px solid #999999;
    background: #f0f0f0;
    height: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:horizontal {
    background: #666666;
    border: 1px solid #666666;
    min-width: 20px;
}

QScrollBar::add-line:horizontal {
    border: 1px solid #999999;
    background: #f0f0f0;
    width: 0px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
    border: 1px solid #999999;
    background: #f0f0f0;
    width: 0px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::add-page:horizontal,
QScrollBar::sub-page:horizontal {
    background: #4b4b4b;
}

QScrollBar::left-arrow:horizontal,
QScrollBar::right-arrow:horizontal {
    border: 1px solid #999999;
    width: 3px;
    height: 3px;
    background: #f0f0f0;
}
</string>
            </property>
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="sliderPosition">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
             <string>Задержка ПКМ</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="rbutton_wait_label">
            <property name="text">
             <string>мс</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>130</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #484848;
border: 2px solid #3e3e3e;
border-radius: 5px;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="own_click">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Указывает, сколько кликов будет выполнено дополнительно за одно нажатие клавиши мыши&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox {
	border: none;
    color: white;
}

QCheckBox::indicator {
    background-color: #333; 
    border: 1px solid #555; 
    width: 15px; 
    height: 15px; 
}

QCheckBox::indicator:checked {
    background-color: #008080;
}</string>
         </property>
         <property name="text">
          <string>Вкл/выкл свой КПС за клик (?)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="title">
          <string>Укажите желаемые значения</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Правая кнопка мыши</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Левая кнопка мыши</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="lbutton_click_counter">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="rbutton_click_counter">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
